src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/DataFrameTests.cs

```diff
@@ -680,10 +680,15 @@ public void TestSignaturesV2_4_X()
 
             {
                 RelationalGroupedDataset df = _df.GroupBy("name");
+                var values = new List<object> { 19 };

```

Comment:
Can we test with multiple values with different types?

Reply:
Did you mean something like this?
`var values = new List<object> { 19, "twenty" };`